tests/serverDescription/serverDescription-getHelloResponse-001.phpt

```diff
@@ -10,13 +10,12 @@ require_once __DIR__ . "/../utils/basic.inc";
 $manager = create_test_manager();
 $server = $manager->selectServer(new MongoDB\Driver\ReadPreference('primary'));
 
-var_dump($server->getServerDescription()->getHelloResponse());
+$helloResponse = $server->getServerDescription()->getHelloResponse();
+var_dump(array_key_exists('ismaster', $helloResponse) || array_key_exists('isWritablePrimary', $helloResponse));
```

Comment:
I've changed this for a more detailed inspection (to ensure that we're not receiving an empty array). Since the response could be anything, I opted to check for a single field that we know is there.